recipe/migrations/root620000.yaml

```diff
@@ -1,15 +1,11 @@
-migrator_ts: 1583150223
+migrator_ts: 1583251649
 __migrator:
   kind:
     version
   migration_number:
-    1
+    2
```

Comment:
We can keep the timestamp and migration number the same I think. 

@chrisburr for double checking this

Reply:
Yes we can